templates/template/maintenanceMode.html.twig

```diff
@@ -0,0 +1,10 @@
+{% extends "template/UI.html.twig" %}
+
+{% block title %}
+    System is in read-only mode
```

Comment:
Shouldn't this title also be "Maintenance Mode" ?

Reply:
Fixed it